libs/wingsdk/src/target-sim/bucket.inflight.ts

```diff
@@ -42,7 +44,39 @@ export class Bucket implements IBucketClient, ISimulatorResourceInstance {
     return this._fileDir;
   }
 
-  public async init(): Promise<BucketAttributes> {
+  public async init(dir?: string): Promise<BucketAttributes> {
```

Comment:
Wouldn't it make more sense to simply use the state directory as the backing store of the bucket and then we persist continuously instead of snapshots?

Reply:
Updated